src/coreclr/ildasm/windasm.cpp

```diff
@@ -424,37 +423,10 @@ int ProcessOneArg(_In_ __nullterminated char* szArg, _Out_ char** ppszObjFileNam
     return 0;
 }
 
-char* UTF8toANSI(_In_ __nullterminated char* szUTF)
```

Comment:
Instead of doing all these complicated conversions, we just use the argc+argv from the standard C main or wmain methods.